src/bp/cluster.ts

```diff
@@ -68,5 +82,31 @@ export const setupMasterNode = (logger: sdk.Logger) => {
     }
   })
 
-  cluster.fork({ SERVER_ID: process.SERVER_ID })
+  spawnWebWorker()
+}
+
+function spawnWebWorker() {
+  const { id } = cluster.fork({ SERVER_ID: process.SERVER_ID, WORKER_TYPE: <WorkerType>'WEB_WORKER' })
+  process.WEB_WORKER = id
+}
+
+let spawnMLWorkersCount = 0
+setTimeout(() => {
+  spawnMLWorkersCount = 0
+}, ms('2m'))
+
+export async function spawnMLWorkers(logger?: sdk.Logger) {
+  if (spawnMLWorkersCount > 2) {
```

Comment:
The variable isn't clear enough, maybe mlWorkersRebootCount ? So it match the one for web worker